src/main/java/Game.java

```diff
@@ -0,0 +1,104 @@
+public class Game {
+	
+	private Player[] gamePlayer;
+	private static Map map;
+	private static java.util.Date timestamp;
+
+	Game(){
+		gamePlayer = new Player[4];
+		map = new Map();
+		timestamp = new java.util.Date();
+	}
+	
+	void startGame() {
+		//For the purpose of this game, we will be using only 1 player. multiplayer is easily expandable.
+		// If not, for each player you would have to open a new thread and window for them to play. OR network it.
+		gamePlayer[0] = new Player(map.getEntryPoint());
+		runPlayer(gamePlayer[0]);
+	}
+	
+	public static void main(String args[]) {
+		new Game().startGame(); //Only 1 player
+	}
+
+	public static final void runPlayer(Player player) {
+		//LoginUserName
+		player.output("To join the VuORK realm, you need some sort of identification. So please, What is your name?");
+		player.setName(player.getLine());
+		player.output("Welcome " + player.getName() + " to the world of VuOrk! <Print Help by typing 'Help'>\n======================================\n");
+		player.output(map.getDescription(player.position()));
+
+		//Loop prompt
+		while(true) {
+			Instruction currCommand = player.getInstruction(); //Blocking command waiting for input
+			validateCommand(currCommand, player); //validateCommand
+		}
+	}
+
+	public static void validateCommand(Instruction command, Player player) {
+		String action = command.getAction();
+		//Movement shortcut
+		if (isDirection(action)) {handleMove(action, player); return;}
+		switch(action) {
+			case "move":
+				handleMove(command.getItems().get(0), player);
+				break;
```

Comment:
Validating the player that has given the command seems unnecessary in a single player game.